File: Quizzes/q57.c
#include <stdio.h>

int fibonacci(int index)
{
  if (index <= 0)
  {
    return 0;
  }
  else if (index == 1)
  {
    return 1;
  }
  else
  {
    return fibonacci(index-1) + fibonacci(index-2);
  }
}

int main()
{
  int index;

  printf("Enter an index for the Fibonacci sequence: ");
  scanf("%d", &index);

  int value = fibonacci(index);
  printf("Fibonacci(%d) = %d\n", index, value);

  return 0;
}

File: Quizzes/q57.py
def fibonacci(index):
  if index <= 0:
    return 0
  elif index == 1:
    return 1
  a,b,temp = 0,1,0
  for x in range(2, index+1):
    temp = a+b
    a = b
    b = temp
  return b

if __name__ == "__main__":
  index = int(input("Enter an index for the Fibonacci sequence: "))
  print(f"Fibonacci({index}) = {fibonacci(index)}")
